temporal/default_failure_converter.go

```diff
@@ -0,0 +1,42 @@
+// The MIT License
```

Comment:
I don't understand? We can't move this to `converter` because this file has dependencies on `internal`. If we move it we will have a circular dependency.

Reply:
Oh, my mistake! Ignore me, sorry about that.